src/main/java/org/icgc/argo/workflow_management/streams/WesConsumerConfig.java

```diff
@@ -86,10 +81,10 @@ private Disposable createWfMgmtRunMsgForExecuteConsumer() {
       val msg = tx.get();
       log.debug("WfMgmtRunMsg received: {}", msg);
 
-      if ((initializingByMiddleware.equals(false) && msg.getState().equals(RunState.QUEUED))
-          || (initializingByMiddleware.equals(true) && msg.getState().equals(RunState.INITIALIZING))) {
+      if (msg.getState().equals(RunState.INITIALIZING)) {
```

Comment:
I removed this logic... it fdidn't feel right here. I moved it in to workflow-search/api instead since it already has to have knowledge of what exchange/queue to send msgs  to.. might as well set the proper state.